symphony-bdk-http/symphony-bdk-http-jersey2/src/main/java/com/symphony/bdk/http/jersey2/ApiClientJersey2.java

```diff
@@ -340,32 +342,59 @@ protected boolean isJsonMime(String mime) {
    * @return Entity
    */
   protected Entity<?> serialize(Object obj, Map<String, Object> formParams, String contentType) {
-    Entity<?> entity;
-    if (contentType.startsWith("multipart/form-data")) {
-      MultiPart multiPart = new MultiPart();
-      for (Entry<String, Object> param : formParams.entrySet()) {
-        if (param.getValue() instanceof File) {
-          File file = (File) param.getValue();
-          FormDataContentDisposition contentDisp = FormDataContentDisposition.name(param.getKey())
-              .fileName(file.getName()).size(file.length()).build();
-          multiPart.bodyPart(new FormDataBodyPart(contentDisp, file, MediaType.APPLICATION_OCTET_STREAM_TYPE));
-        } else {
-          FormDataContentDisposition contentDisp = FormDataContentDisposition.name(param.getKey()).build();
-          multiPart.bodyPart(new FormDataBodyPart(contentDisp, parameterToString(param.getValue())));
+    if (contentType.startsWith(MediaType.MULTIPART_FORM_DATA_TYPE.toString())) {
+      return this.serializeMultiPartFormDataEntity(formParams);
+    }
+    else if (contentType.startsWith(MediaType.APPLICATION_FORM_URLENCODED_TYPE.toString())) {
```

Comment:
same here for `MediaType.APPLICATION_FORM_URLENCODED`

Reply:
Fixed